engine/src/flutter/shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -184,13 +148,27 @@ new PlatformMessageHandlerIos(task_runners.GetPlatformTaskRunner())) {}
 void PlatformViewIOS::UpdateSemantics(flutter::SemanticsNodeUpdates update,
                                       flutter::CustomAccessibilityActionUpdates actions) {
   FML_DCHECK(owner_controller_);
+  FML_DCHECK(accessibility_bridge_);
   if (accessibility_bridge_) {
     accessibility_bridge_.get()->UpdateSemantics(std::move(update), actions);
     [[NSNotificationCenter defaultCenter] postNotificationName:FlutterSemanticsUpdateNotification
                                                         object:owner_controller_];
   }
 }
 
+// |PlatformView|
+void PlatformViewIOS::SetSemanticsTreeEnabled(bool enabled) {
+  FML_DCHECK(owner_controller_);
+  if (enabled) {
+    if (!accessibility_bridge_) {
+      accessibility_bridge_ = std::make_unique<AccessibilityBridge>(owner_controller_, this,
+                                                                    platform_views_controller_);
+    }
```

Comment:
Super minor style nit, but I'd be tempted to write this as a bailout return just to make it a a super tiny bit easier to follow.
```cc
if (accessibility_bridge) {
  return;
}
accessibility_bridge = ...
```